public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/sessions.js

```diff
@@ -0,0 +1,168 @@
+}
+
+// todo individual session
+function Session( { session } ) {
+	const { id, link, slug, title }      = session;
+	const { startTime, endTime, tracks } = session.derived;
+	const speakers                       = session._embedded.speakers || [];
+
+	/*
+	 * These are _only_ opened in new tabs because this is in the editor. They should _not_ do that on the front-end,
+	 * because that's an anti-pattern that hurts UX by taking control away from the user.
+	 *
+	 * todo move ^ comment below with session-title if this one moves down there too?
+	 */
+
+	const classes = classnames(
+		'wordcamp-schedule-session',
+		`wordcamp-schedule-session-type-${ session.meta._wcpt_session_type }`
+
+		// change all class names everywhere in this file, but wait until make dynamic, since it'll take a lot less time then
+
+		// if it's just these 2, then probably don't need classnames();
+	);
+
+	//console.log(tracks[0][0]); // wtf why nested ?
+
+	// if lunch etc then
+	//      apply wordcamp-schedule-session-type-custom class instead of wordcamp-schedule-track-{id}. maybe apply all of the `track-{id}` classes? probably.
+	//          use data- attrs instead of classes?
+	//          only apply if crosses all of them? no, probably even if just crosses 2 or more
+	//      no author, category, etc. those will maybe happen automatically b/c only showning if they're assigned? might want category in some cases anyway
+
+	const gridColumn = `wordcamp-schedule-track-${ tracks[0].id }`;  // todo tmp, need to account for multiple
+
+		// todo how to get track id? maybe add data-slug to the heading, and then lookup the value of the heading where the slug matches this one?
+			// have to account for spanning multiple _contiguous_ tracks, looks like: grid-column: track-3-start / track-4-end;
+				// don't have to support non-contiguous
+				// BUT, how are you supposed to programatically determine if they're contiguous?
+				// maybe just don't support that spanning?
+					// has to either be in 1 track or all of them (e.g., lunch), but not 2? does the current schedule support spanning tracks?
+				// maybe have if ( type='custom' ) then { ...}  else { output using id } ?
+			// if don't support continugoius spanning automatically, could at least leave a note for organizers that they can do it manually if they want
+		// todo would this be easier to populate with slugs/ids instead of ints? maybe, but maybe other parts wouldn't?
+
+	const gridRow = `
+		time-${ format( 'Hi', startTime ) } /
+		time-${ format( 'Hi',   endTime ) }
+	`;
+
+	return (
+		<div
+			className={ classes }
+			data-wordcamp-schedule-session-id={ id }
+			data-wordcamp-schedule-session-slug={ slug }
+			data-wordcamp-schedule-session-tracks={ tracks.map( track => track.slug ) }
+				// test w/ 0, 1, and 2 tracks
+				// need space or comma or something inbetween? think about how they'll be targeted w/ css
+			style={ { gridColumn, gridRow } }
+		>
+			<h4 className="wordcamp-schedule-session-title">
```

Comment:
Same point as above, we shouldn't be this opinionated about heading levels, and I don't think that these need to be headings. I'll take a look at what seems to be best practices around this for a11y.